### ISSUE TITLE
`oneOf` not working inside `properties`

### ISSUE THREAD

**Issue by FlorianLudwig** [koxudaxi/datamodel-code-generator#139]
**Describe the bug**
I think it is allowed to use `oneOf` inside `properties` but it won't work.

**To Reproduce**

small.json
```json
{
    "definitions": {
      "io.k8s.api.admissionregistration.v1.MutatingWebhook": {
        "properties": {
          "timeout": {
            "oneOf": [
                {
                  "type": "string"
                },
                {
                  "type": "integer"
                }
            ]
          }
        },  
        "type": "object"
      }
    }
}
```

```bash
$ datamodel-codegen --input small.json --input-file-type jsonschema 
Traceback (most recent call last):
  [...]
  File "datamodel_code_generator/parser/jsonschema.py", line 158, in get_data_type
    raise ValueError(f"invalid schema object {obj}")
ValueError: invalid schema object items=None uniqueItem=None type=None format=None pattern=None minLength=None maxLength=None minimum=None maximum=None multipleOf=None exclusiveMaximum=None exclusiveMinimum=None additionalProperties=None anyOf=[] allOf=[] enum=[] writeOnly=None properties=None required=[] ref=None nullable=False x_enum_varnames=[] description=None title=None example=None examples=None default=None
```

**Comment by koxudaxi** [comment]
@FlorianLudwig

Thank you for your report.
Yes, the feature is not implemented yet.
I think the generator should create validation method for `oneOf`.
I will implement it. I hope to try this week

**Comment by koxudaxi** [comment]
I think we should resolve the problem(https://github.com/koxudaxi/datamodel-code-generator/issues/142) for this issue at first.

**Comment by koxudaxi** [comment]
I  have released a new version as `0.5.6`
